packages/cli/src/commands/composite/create.ts

```diff
@@ -30,19 +27,17 @@ export default class CreateComposite extends Command<Flags, { schemaFilePath: st
   }
 
   async run(): Promise<void> {
-    this.spinner.start('Creating the composite...')
     try {
-      const schema = await fs.readFile(resolvePath(this.args.schemaFilePath), { encoding: 'utf-8' })
-      const composite = await Composite.create({ ceramic: this.ceramic, schema })
-      const encodedAsJSON = JSON.stringify(composite.toJSON(), null, 4)
+      const composite = await createComposite(this.ceramic, this.args.schemaFilePath)
       if (this.flags.output !== undefined) {
         const output = this.flags.output
-        await write(output, encodedAsJSON)
+        await writeEncodedComposite(composite, output)
         this.spinner.succeed(
           `Composite was created and its encoded representation was saved in ${output}`
         )
       } else {
-        this.spinner.succeed(encodedAsJSON)
+        // Not using the spinner here, so that the output can be piped using standard I/O
+        console.log(JSON.stringify(composite.toJSON(), null, 2))
```

Comment:
Please avoid formatting the JSON output, we don't know what the output can be piped to so let's keep it as neutral as possible.
```suggestion
        this.log(JSON.stringify(composite.toJSON()))
```